common/Latin.yml

```
@@ -6,5 +6,7 @@ nonword: true
 action:
   name: replace
 swap:
-  '\b(?:eg|e\.g\.)[\s,]': for example
-  '\b(?:ie|i\.e\.)[\s,]': that is
+  '\b(?:eg|e\.\s?g\.)\s+(?![,\.;])': 'for example '
+  '\b(?:eg|e\.\s?g\.)\s*[,\.;]': 'for example'
+  '\b(?:ie|i\.\s?e\.)\s+(?![,\.;])': 'that is '
+  '\b(?:ie|i\.\s?e\.)\s*[,\.;]': 'that is' 
```

you have to prefix ie (and eg) with a [negative look behind](https://www.regular-expressions.info/lookaround.html)

Said otherwise, the character before ie cannot be a dot

```suggestion
  '\b(?:(?<!.)eg|e\.\s?g\.)\s+(?![,\.;])': 'for example '
  '\b(?:(?<!.)eg|e\.\s?g\.)\s*[,\.;]': 'for example'
  '\b(?:(?<!.)ie|i\.\s?e\.)\s+(?![,\.;])': 'that is '
  '\b(?:(?<!.)ie|i\.\s?e\.)\s*[,\.;]': 'that is' 
```

https://regexr.com/8239f

regexp are so fun an easy to read

---

The new rules miss out on the last 2 sections of the test file.